Issues/45 - Accordion 'after-content' media mobile fixes

smcgrath0 opened the pull request:
**What does this implement/fix? Explain your changes.**

Accordion's 'after-content' media was not working for mobile displays. 

**Does this close any currently open issues?**

#45 

Added more to hatch.js because previously wasn't hatched.

**Where has this been tested?**
Local

<img width="451" alt="Screen Shot 2020-08-11 at 4 58 51 PM" src="https://user-images.githubusercontent.com/49845255/89960587-4a1ae080-dbf4-11ea-9104-11d2a405d9d1.png">


*Browser (version):* Chrome v84

smcgrath0 committed:
#45 accordian works, need to add in hatch

smcgrath0 committed:
#45 accordion fixed with hatch

smcgrath0 committed:
#45 fixed unhatched attribute from previous contributor

reusr1 commented on fragments/accordion/hatch.js:
```diff
@@ -6,9 +6,12 @@ module.exports = {
         //Header
     	f.addIf($.find('h2').first(), 'model.showtitle == \'true\'')
         f.mapField($.find('h2').first(), "model.title")
+        f.bindAttribute($.find('h2').first(), "data-per-inline", "`model.title`")
```
this should not be necessary, please upgrade to the latest percli version - mapField and mapRichField are adding data-per-inline automatically in the latest version

reusr1 reviewed (changes requested):
see comments above

smcgrath0 committed:
Merge branch 'develop' of https://github.com/headwirecom/themeclean-flex into issues/45

smcgrath0 committed:
#45 upgraded percli and removed unneeded attributes

smcgrath0 committed:
#45 fixed accordion theme colors to work on all sites

smcgrath0 committed:
#45 another unneeded attr bc of percli upgrade

smcgrath0 commented:
Upgraded my percli and removed the attributes mentioned. 

**ADDITION**
I also found a minor theme color bug with the accordion component

Bug:
<img width="1115" alt="Accordion-background-test" src="https://user-images.githubusercontent.com/49845255/90940522-efd40980-e3c3-11ea-9f00-cf194ca87bf3.png">

Fixed:
<img width="1112" alt="accordion-background-fixed" src="https://user-images.githubusercontent.com/49845255/90940599-2e69c400-e3c4-11ea-8031-f5c3443130f6.png">
